rct229/rulesets/ashrae9012019/section23/section23rule4.py

```diff
@@ -65,9 +65,10 @@ def get_calc_vals(self, context, data=None):
             baseline_system_types_dict = get_baseline_system_types(rmd_b)
 
             hvac_sys_5_or_7_list = [
-                system_type
+                value
                 for system_type in baseline_system_types_dict
                 for applicable_sys_type in APPLICABLE_SYS_TYPES
+                for value in baseline_system_types_dict[system_type]
                 if baseline_system_type_compare(system_type, applicable_sys_type, False)
             ]
```

Comment:
Will this result in 2D array that needs to be flattened?

Reply:
No it results in a list of system ID's instead of a list of system type strings, so that the applicability check has a chance of finding hvac_sys_id within the list.

Reply:
Make sense.